sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/MerchantOrder.java

```diff
@@ -101,7 +114,9 @@ public CustomerInformation getCustomerInformation() {
         private CustomerInformation customerInformation;
         private PaymentBrand paymentBrand;
         private PaymentBrandForce paymentBrandForce;
+        private Map<String, String> paymentBrandMetaData = Collections.emptyMap();
         private String initiatingParty;
+        private boolean skipHppResultPage;
```

Comment:
For clarity, would it be an idea to set the default value to false? (It is the implicit default value, but maybe it is good to make it explicit)